### PR TITLE
Html

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -11,11 +11,12 @@ import (
...
-		etcdPeers  = flag.String("etcd-peers", "http://localhost:4001", "Comma-separated list of addresses of etcd endpoints to connect to")
-		keyPrefix  = flag.String("key-prefix", "/vulcand/frontends/", "Key prefix to list of services in etcd")
-		vulcand    = flag.String("vulcand", "localhost:8080", "Vulcand address")
-		exclude    = flag.String("exclude", "", "Comma-separated list of services to exclude from healthcheck")
+		socksProxy  = flag.String("socks-proxy", "", "Use specified SOCKS proxy (e.g. localhost:2323)")
+		etcdPeers   = flag.String("etcd-peers", "http://localhost:4001", "Comma-separated list of addresses of etcd endpoints to connect to")
+		keyPrefix   = flag.String("key-prefix", "/vulcand/frontends/", "Key prefix to list of services in etcd")
+		vulcand     = flag.String("vulcand", "localhost:8080", "Vulcand address")
+		exclude     = flag.String("exclude", "", "Comma-separated list of services to exclude from healthcheck")
+		elbHostname = flag.String("elb-hostname", "", "Public hostname - cluster entrypoint")
```
Comment (martingartonft) [inline main.go]: maybe call this just "hostname"? it might not be an elb

**Comment by martingartonft** [comment]
looks good overall. wondering whether it makes sense to push any of the html-ability of this into the health check library?
We could always do that as a later step though.

**Comment by catalin-ilea** [comment]
@Nesze is adding html-ability to the go and java healthcheck libraries as we speak

**Comment by Zapadlo** [comment]
+2

File: main.go (hunk 1 above)
Comment (catalin-ilea) [inline main.go]: Done

**Pull request merged**